File: ad-hoc/javascript/3465.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('', input => {
  const [a, b, c] = input.split(' ').map(Number);

  const s = (a + b + c) / 2;
  const area = Math.sqrt(s * (s - a) * (s - b) * (s - c));

  console.log(area.toFixed(3) + ' m2');

  rl.close();
});

File: ad-hoc/python/3465.py
import math

def main():
    a, b, c = map(float, input().split())

    s = (a + b + c) / 2.0
    area = math.sqrt(s * (s - a) * (s - b) * (s - c))

    print("{:.3f} m2".format(area))

if __name__ == "__main__":
    main()
